### PR TITLE
set @capacitor/core to version 2.4.7 or higher

### PR DESCRIPTION
**Pull request by tobium**
fixed capacitor 3 problem (plugin uses capacitor 3 but is not compatible with cap3)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@capacitor-community/firebase-analytics",
-  "version": "0.3.0",
+  "version": "0.3.1",
```
Comment (brownoxford) [inline package.json]: Please don't bump the version - this happens during release.

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   "author": "mesur.io <oss@mesur.io>",
   "license": "MIT",
   "dependencies": {
-    "@capacitor/core": "latest"
+    "@capacitor/core": "^2.4.7"
```
Comment (brownoxford) [inline package.json:19]: I believe `^2.4.7` is also required for dev deps on @capacitor/android and @capacitor/ios

Comment (tobium) [reply package.json:19]: thx, changed it

File: package.json (hunk 1 above)
Comment (tobium) [reply package.json]: changed it bacj

**Comment by tobium** [comment]
pls review again

**Review by slajar — APPROVED** [review]

**Review by brownoxford — APPROVED** [review]
Looks great, thank you for the patch!

**Pull request merged**